### ISSUE TITLE
Fallbacks dont work as expected

### ISSUE THREAD

**Issue by thepassle** [guybedford/es-module-shims#31]
For import map:

```
    <script type="importmap-shim">
      {
        "imports": {
          "@vaadin/router": "/web_modules/@vaadin/router.js",
          "lit-element": "/web_modules/lit-element.js",
          "/web_modules/kv-storage-polyfill.js": [
            "std:kv-storage",
            "/web_modules/kv-storage-polyfill.js"
          ]
        }
      }
    </script>
```

results in:

```
es-module-shims.js:40 Uncaught TypeError: Cannot read property 'slice' of undefined
    at resolveIfNotPlainOrUrl (es-module-shims.js:40)
    at resolvePackages (es-module-shims.js:131)
    at parseImportMap (es-module-shims.js:138)
    at es-module-shims.js:966
    at es-module-shims.js:993
```

It seems to be related to this line:
https://github.com/guybedford/es-module-shims/blob/master/dist/es-module-shims.js#L130

`resolveIfNotPlainOrUrl` takes 2 arguments, `relUrl` and `parentUrl`:
```js
function resolveIfNotPlainOrUrl (relUrl, parentUrl) {
```

But on the [line linked above](https://github.com/guybedford/es-module-shims/blob/master/dist/es-module-shims.js#L130), it only takes one arg: 
```js
outPkgs[resolveIfNotPlainOrUrl(p) || p] = value;
```

So obviously `parentUrl.slice` isn't gonna work when its undefined. As a dirty workaround, changing the line to 
```js
outPkgs[resolveIfNotPlainOrUrl(p, p) || p] = value;
``` 

Does work.

Also changing the import map to:

```
      {
        "imports": {
          "@vaadin/router": "/web_modules/@vaadin/router.js",
          "lit-element": "/web_modules/lit-element.js",
          "std:kv-storage": [
            "std:kv-storage",
            "/web_modules/kv-storage-polyfill.js"
          ]
        }
      }
```

Also works fine, but this results in an error in the chrome dev console (chrome):

```
Ignored an import map key "std:kv-storage": Invalid key (non-fetch scheme)
```

Which means I guess that the spec doesnt actually support this?

I'll try and see if I have some time today to investigate further and whip up a PR. If you have any pointers I'd be happy to hear 😊

**Comment by guybedford** [comment]
Thanks, this should be fixed in https://github.com/guybedford/es-module-shims/pull/32. Released in 0.2.15.

**Comment by guybedford** [comment]
Btw I think you need to ensure you are running Chrome canary with the `Experimental Web Platform` flag enabled to get `std:kv-storage`. Do let me know if it's still not working though.
